mmpose/models/detectors/top_down.py

```diff
@@ -211,7 +206,8 @@ def forward_dummy(self, img):
 
     def process_head(self, output, img, img_metas, return_heatmap=False):
         """Process heatmap and keypoints from backbone features."""
-        flip_pairs = img_metas['flip_pairs']
+        flip_pairs = img_metas[0]['flip_pairs']
+        batch_size = img.size(0)
```

Comment:
Should `bbox_id` be mandatory if batch_size > 1?

Reply:
I'm thinking what happens if a new Dataset implementer forgot to inject `bbox_id`